.solcover.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
     skipFiles: [
       'Migrations.sol',
       'EtherRouter.sol',
-      'gnosis'
+      'gnosis',
+      'PatriciaTree'
```

Comment:
The source isn't really maintained or tested. Either we test it here or we submit that to source repo